pkg/transformer/openshift/openshift.go

```diff
@@ -81,6 +92,52 @@ func initImageStream(name string, service kobject.ServiceConfig) *imageapi.Image
 	return is
 }
 
+// initBuildConfig initialize Openshifts BuildConfig Object
+func initBuildConfig(name string, service kobject.ServiceConfig) *buildapi.BuildConfig {
+	bc := &buildapi.BuildConfig{
+		TypeMeta: unversioned.TypeMeta{
+			Kind:       "BuildConfig",
+			APIVersion: "v1",
+		},
+		ObjectMeta: api.ObjectMeta{
+			Name: name,
+		},
+		Spec: buildapi.BuildConfigSpec{
+			// Triggers
+			[]buildapi.BuildTriggerPolicy{
+				{Type: "ConfigChange"},
+				{Type: "ImageChange"},
+			},
+			// RunPolicy
+			"serial",
+			buildapi.CommonSpec{
+				Source: buildapi.BuildSource{
+					Git: &buildapi.GitBuildSource{
+						Ref: "master",
+						URI: getGitRemote("origin"),
+					},
+					ContextDir: "./",
```

Comment:
The value for ``ContextDir`` cannot be value of ``build`` or ``build.context`` from Docker Compose file. There may be cases where the compose file is not at the root directory of the project. In that case, we evaluate the path relative from the root directory of the project, instead, of the relative path from the compose file.